src/main/java/run/halo/app/config/WebServerSecurityConfig.java

```diff
@@ -48,6 +49,7 @@
  *
  * @author johnniang
  */
+@Configuration
```

Comment:
Why do I add `@Configuration` annotation at here?

Please see https://github.com/spring-projects/spring-security/issues/6613 for more.